src/main/java/com/google/devtools/common/options/OptionsClassProvider.java

```diff
@@ -19,6 +19,7 @@
  * A read-only interface for options parser results, which only allows to query the options of
  * a specific class, but not e.g. the residue any other information pertaining to the command line.
  */
+@FunctionalInterface
```

Comment:
Okey dokey.

In that case, should the `@FunctionalInterface` lines that I added to other interfaces in this PR stay?